src/vehicle_gui.cpp

```diff
@@ -438,10 +438,7 @@ struct RefitWindow : public Window {
 				bool first_vehicle = this->list[current_index].size() == 0;
 				if (first_vehicle) {
 					/* Keeping the current subtype is always an option. It also serves as the option in case of no subtypes */
-					RefitOption *option = this->list[current_index].Append();
-					option->cargo   = cid;
-					option->subtype = 0xFF;
-					option->string  = STR_EMPTY;
+					this->list[current_index].push_back({cid, 0xFF, STR_EMPTY});
```

Comment:
Would `emplace_back(cid, 0xFF, STR_EMPTY)` work here?

Reply:
Unfortunately not